src/main/java/ch/njol/skript/SkriptConfig.java

```diff
@@ -50,230 +34,185 @@
 import ch.njol.skript.util.chat.ChatMessages;
 import ch.njol.skript.util.chat.LinkParseMode;
 import ch.njol.skript.variables.Variables;
-import ch.njol.util.Setter;
+import org.bukkit.event.EventPriority;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collection;
 
 /**
  * Important: don't save values from the config, a '/skript reload config/configs/all' won't work correctly otherwise!
- * 
+ *
  * @author Peter Güttinger
  */
 @SuppressWarnings("unused")
 public abstract class SkriptConfig {
-	private SkriptConfig() {}
-	
+	private SkriptConfig() {
+	}
+
 	@Nullable
-	static Config mainConfig;
-	static Collection<Config> configs = new ArrayList<Config>();
-	
-	final static Option<String> version = new Option<String>("version", Skript.getVersion().toString())
+	private static Config mainConfig;

```

Comment:
Probably intentional to have it package-private. Or is there getter somewhere?

Reply:
Reverted